### PR TITLE
Category filter styling

### PR DESCRIPTION
**Pull request by gkalvatcheva**
• replaced some values with variables
• variables for icon size and font weight for subcategory filters
• width and height adjustments for open down vs open up panel
• removed entire .above section from expanding when we open the
subcategories panel

### CODE REVIEW

File: app/scripts/components/dashboard/charts-container.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes } from 'react';
```
Comment (esoliani) [inline app/scripts/components/dashboard/charts-container.jsx]: have you done any change to this file. it should not be used.

File: app/scripts/components/dashboard/sub-category-filter.jsx [hunk 2]
```diff
@@ -56,6 +56,7 @@ const SubCategoryFilter = React.createClass({
     return (
       <div className="sub-category-filter">
         <div className="category-collapsable" onClick={this.toggle}>
+          <T k={`charts.sub-category.${this.props.type}`} />&nbsp;<Icon type={`chevron-${direction}`}/>
```
Comment (esoliani) [inline app/scripts/components/dashboard/sub-category-filter.jsx:59]: this is duplicating the categories.wrong!

Comment (galinagk) [inline app/scripts/components/dashboard/sub-category-filter.jsx:59]: Mmm, I change the arrow icon only here in icon type.

Comment (esoliani) [inline app/scripts/components/dashboard/sub-category-filter.jsx:59]: maybe it's a merge issue

File: app/scripts/components/dashboard/charts-container.jsx (hunk 1 above)
Comment (esoliani) [inline app/scripts/components/dashboard/charts-container.jsx]: Please remove this file, so we don't misunderstand again

**Pull request merged**